### ISSUE TITLE
Consumer Protection Data Analysis

### ISSUE THREAD

**Issue by mateoclarke** [open-austin/project-ideas#107]
### What problem are we trying to solve?

Staff at the Innovation Office are trying to compile and analyze available data about consumer protection and complaints. Currently, there are these two datasets attached as Excel files. This data is incomplete and not very detailed but it is all we have at this point to help us build a case for why Cities for Financial Empowerment should make a huge investment in Austin. We want to know about the depth and scale of the problem. We also have some narrative stories to help build the case.

[2016 211 Travis County Caller Needs.xlsx](https://github.com/open-austin/project-ideas/files/1282103/2016.211.Travis.County.Caller.Needs.xlsx)
[Consumer_Complaints Austin.xlsx](https://github.com/open-austin/project-ideas/files/1282102/Consumer_Complaints.Austin.xlsx)

### Who will benefit (directly and indirectly) from this project?

Families and Individuals in Austin area vulnerable to high interest loans and predatory financial products.

City of Austin will benefit by having more resources and insights in addressing services and policies in this domain.

### Where can we find any research/data available/articles?

Here is more information about the [Consumer Financial Protection Austin grant DRAFT](https://docs.google.com/document/d/1ZeY-q46qv5cISXVHT_irFh45OkL6PW7Ib0oBj1yZRSY/edit?usp=sharing)

Datasets linked above. One is consumer complaint data from CFPB (787xx zip codes) and the other is 211 data. ⬆️ 

### What help is needed at this time?

We would love to work with anyone interested in data analysis and basic visualization of the two datasets attached. 

No preference on tools or presentation. Anything from Excel to Tableau to R to Python to d3.js would be very cool.

Please respond to this thread and/or email us if you'd like to help or ask any questions: 
<a href="mailto:angela.hanson@austintexas.gov; farah.muscadin@austintexas.gov; mateo.clarke@austintexas.gov">Angela, Farah, and Mateo and CoA Innovation Office</a>

### What are the next steps (validation, research, coding, design)?

Data analysis & Data viz

----

#### Checklist for NEW ideas :baby:
Hey, you're official! You're now part of the growing civic hacking community in Austin. Here's a few things to get started (a couple you've probably already done).

- [x] Create this idea issue
- [x] Flesh out the who, where, and what questions above
- [x] Start the conversation about this new idea on [Slack](https://openaustin-slackin.herokuapp.com/)!
- [x] Respond with at least one update on this issue within the next month

#### Checklist for ACTIVE projects :fire:
Let's get this project started! When this idea starts taking off, the Projects Core Team will start helping this project's lead(s) out with project management and connecting you to resources you may need. To get there, please complete and check off the following:

- [ ] Complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101)
- [x] Create a [GitHub repository](https://github.com/new) and Slack channel for work.
- [x] Create a [README file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/) in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
- [ ] [Create issues](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/project-set-up-for-collaboration-with-github/#assignment--add-your-first-issue) to describe each task that you plan to do or need help with and how a contributor can get started on that task. You might start and stop a lot, so consider issues as your to-do list.

#### Checklist for FEATURED Projects :tada:
To have your project FEATURED on [Open-Austin.org](https://open-austin.org/), complete the following documentation. In past projects, well-documented featured projects have more contributions than other projects.

* [ ] In your README, link to the [Open Austin Community Participation Guidelines](https://docs.google.com/document/d/1OujyBccPpepUXSY5_nP3alzvd81WlCKSeSf_-sU5K-U/edit?usp=sharing) or write your own code of conduct.
* [x] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
* [ ] [Create file: CONTRIBUTING.md](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md) so others know [how they can contribute](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/).

Once all of the above is complete,
- [ ] [Create an issue](https://github.com/open-austin/open-austin.github.io/issues/new) on the open-austin.github.io repo with the title `Add [this project] to projects page. An Open Austin leader will review this issue and post your project :balloon:

If you get stuck at any point, feel free to reach out to the leadership team on Slack by adding @leadership to your message. We're here to help you make real changes to our city.

**Comment by mateoclarke** [comment]
Total number of complaints
·         Trends in types of complaints
·         Trends in complaints by year
·         Trends of complaints by area/zip code
·         Complaints by issue and year
·         Percentage of issues by zip code – a map would be ideal
·         Trends in percentage of resolution (data field)
·         Percentage of resolution (data field) by area/zip code (Where do most unresolved complaints occur?)
·         Trends in complaints by company and product

**Comment by mateoclarke** [comment]
https://github.com/open-austin/consumer-protection

**Comment by mateoclarke** [comment]
Brainstorm of potential solution:
- List of bad actors
- Identify communities or individuals that could be targeted for ethnographic research.
- Within complaints, identify common terms, issues, "hot topics".

**Comment by ecglover8** [comment]
I would like to help with this project. New to Open Austin.

**Comment by fruitschinposamurai** [comment]
@mateoclarke I'm interested in helping with this project- I've been playing around with mapping the 211 financial data. What is the protocol for pushing to the repository? Specifically what branch to push to and if I need to get set up as a collaborator or something?

**Comment by VictoriaODell** [comment]
Thanks for the interest @ecglover8 and @fruitschinposamurai!

This project also has a channel on our slack. Join slack.open-austin.org and it's channel #p-consumer-protection

I've pinged @mateoclarke since I'm unsure the status of this project with some recent changes. Hope to get back to y'all soon.

**Comment by mateoclarke** [comment]
Hey @ecglover8 & @fruitschinposamurai! Thanks for signaling your interest.

Since I posted this project back in September, the City of Austin applied for said the Cities for Financial Empowerment grant but didn't receive the kind of funding required for a full time staff to manage the project. What happens with the limited funding is up in the air while the legal department reviews how the funds get processed. On top of that, the main internal champion, Angela, whom I was working with moved to Paris for an awesome job with the OECD.

All this is to say, we don't have the internal capacity to manage this project and the overall direction is up in the air. 

If anyone of y'all are interested in taking the initiative on this research from a community perspective, I'd be happy to connect you with contacts in the City that might know more about specific programs or initiatives.

**Comment by fruitschinposamurai** [comment]
Thanks @mateoclarke , I'd be happy to still do the research if it's useful. I'm new to the civic hacking scene, so I don't really know how to go about finding someone who wants this research yet.

**Comment by mscarey** [comment]
Hi @AllysonRosenthal, @furuutsuponchisamurai and @mateoclarke, is it okay to close this issue? Did the city ever wrap up this project, or did they have to drop it?

**Comment by mateoclarke** [comment]
They dropped it. We should close this.
